virtcontainers/qemu.go

```diff
@@ -783,7 +783,13 @@ func (q *qemu) waitSandbox(timeout int) error {
 		return fmt.Errorf("Invalid timeout %ds", timeout)
 	}
 
-	cfg := govmmQemu.QMPConfig{Logger: newQMPLogger()}
+	eventCh := make(chan govmmQemu.QMPEvent)
+	cfg := govmmQemu.QMPConfig{
+		EventCh: eventCh,
+		Logger:  newQMPLogger(),
+		// response of qmp.ExecuteQMPCapabilities bigger than default 64k in bufio.NewScanner()
+		MaxCapacity: 512 * 1024,
```

Comment:
can this be a constant ?

Reply:
In most cases, we don't need to modify this value. We only need to keep the default value. In special cases, we need to modify the value according to the actual situation, but we can't find a suitable value to adapt to any situation.

Reply:
I see, the benefit of doing this is that it is convenient to do unit testing, right?
So, do I need to add an additional test case?

Reply:
yes, defining constant is useful for unit testing, not sure if it's possible to add unit tests for this function, but if you can, go for it please